test/integration/sagemaker/util.py

```diff
@@ -124,6 +130,84 @@ def sagemaker_endpoint(sagemaker_client, model_name, instance_type, accelerator_
         sagemaker_client.delete_endpoint_config(EndpointConfigName=model_name)
 
 
+def _create_transform_job_request(model_name, batch_output, batch_input, instance_type):
+    return {
+        'TransformJobName': model_name,
+        'ModelName': model_name,
+        'BatchStrategy': 'MultiRecord',
+        'TransformOutput': {
+            'S3OutputPath': batch_output
+        },
+        'TransformInput': {
+            'DataSource': {
+                'S3DataSource': {
+                    'S3DataType': 'S3Prefix',
+                    'S3Uri': batch_input
+                }
+            },
+            'ContentType': 'text/csv',
+            'SplitType': 'Line',
+            'CompressionType': 'None'
+        },
+        'TransformResources': {
+            'InstanceType': instance_type,
+            'InstanceCount': 1
+        }
+    }
+
+
+def _read_batch_output(boto_session, bucket, model_name):
+    s3 = boto_session.client('s3')
```

Comment:
add the region or else this won't work in GovCloud

Reply:
will do